src/setup.c

```diff
@@ -375,10 +354,65 @@ void setup(void)
 	setup_exceptions();
 	setup_gpio();
 	setup_usart();
-	setup_spi();
 	setup_encoders();
 	setup_pwm();
+	setup_mpu();
 	setup_systick();
 	setup_timer1();
 	setup_adc1();
 }
+
+/**
+ * @brief Setup SPI.
+ *
+ * SPI2 is configured as follows:
+ *
+ * - Master mode.
+ * - Clock baud rate: PCLK1 / speed_div; PCLK1 = 36MHz.
+ * - Clock polarity: 0 (idle low; leading edge is a rising edge).
+ * - Clock phase: 0 (out changes on the trailing edge and input data
+ *   captured on rising edge).
+ * - Data frame format: 8-bits.
+ * - Frame format: MSB first.
+ *
+ * NSS is configured to be managed by software.
+ */
+static void setup_spi(uint8_t speed_div)
```

Comment:
Would you mind moving this to where it was initially (right after `setup_usart()` I think).

Reply:
done